corehq/apps/consumer_user/models.py

```diff
@@ -23,3 +36,20 @@ class ConsumerUserInvitation(models.Model):
     invited_by = models.CharField(max_length=255)
     invited_on = models.DateTimeField(auto_now=True)
     active = models.BooleanField(default=True)
+
+    def make_inactive(self):
+        self.active = False
+        self.save(update_fields=['active'])
+
+    def accept(self):
+        self.accepted = True
+        self.save(update_fields=['accepted'])
+
+    @classmethod
+    def create_invitation(cls, case_id, domain, opened_by, email):
```

Comment:
nit: what's the purpose of this method? In the place you call this, you could just replace this call with `ConsumerUserInviation.objects.create(...)` right?

Reply:
Created this before when code was clumsy